content/sensu-enterprise/3.1/contact-routing.md

```diff
@@ -165,6 +166,49 @@ configuration:
 }
 {{< /highlight >}}
 
+##### Severities
+
+You can use the `severities` attribute to configure the event severities that Sensu Enterprise will route to a contact.
+However, unlike other contact attributes, the `severities` attribute doesn't override `severities` configured in the integration handler definition.
+Sensu Enterprise processes contact-configured severities _after_ integration-configured severities, so contacts can only access severities allowed by the integration configuration.
```

Comment:
Just so I am understanding this right, that means you'd potentially need to have all possible severities under the integration scrope, then filter them at the contact level, to ensure you'd have the choice for all them. Makes me wonder what happens if you leave severities blank on the integration level, then attempt to filter on the contact level if that would work. Seems like it might as the default should be 

Using the example added here, there's no way a contact would be able to get `unknown` alerts as it is not defined in the pagerduty integration with `critical` and `warning`. Is that correct?

Reply:
I am good here. Zoom call we discussed this and I both understand it better and don't believe we need a change here to avoid going off into configuration weeds.

Reply:
Great point. Added https://github.com/sensu/sensu-docs/issues/627 to add defaults for severities at the integration level. To answer your question, yes you would not get unknown alerts if it was explicitly filtered out at the integration level.